tests/Feature/CollectionTest.php

```diff
@@ -113,30 +120,33 @@
     $this->milvus->collections()->create(
         collectionName: 'collection_test',
         dimension: 128,
+        // L2 sets the distance between 0 and ∞ (0 is the closest)
+        metricType: 'L2',
```

Comment:
I can't find any reference that the metric type has changed between versions. But when I ommit this parameter, the distance in the assertion is 1. Might assumption is that the default metric type changed to something like cosine. 

Different metric types and their distnace value ranges: https://milvus.io/docs/metric.md